pkg/lockfile/parse-maven-lock.go

```diff
@@ -243,6 +245,10 @@ func (e MavenLockExtractor) decodeMavenFile(f DepFile, depth int) (*MavenLockFil
 		parentRelativePath = path.Join(parentRelativePath, "pom.xml")
 	}
 	parentPath := filepath.FromSlash(filepath.Join(filepath.Dir(f.Path()), parentRelativePath))
+	if _, err := os.Stat(parentPath); errors.Is(err, os.ErrNotExist) {
+		// If the parent pom does not exist, it still can be in an external repository, but it is unreachable from the parser
```

Comment:
Should this be explicitly printed on stderr?

Reply:
Would be nice to have in case of doubt, adding it :)